packages/lib-react-components/src/ZooFooter/ZooFooter.js

```diff
@@ -90,145 +56,127 @@ const ZooFooter = (props) => {
-            )
-          })}
-        </StyledNavListContainer>
-        <StyledNavListContainer flex margin={{ horizontal: 'none', vertical: 'small' }} tag='nav'>
-          {buildNavListURLs.map((url, i) => {
-            return (
-              <NavListItem colorTheme={colorTheme} key={url} label={buildNavListLabels[i]} url={url} />
-            )
-          })}
-        </StyledNavListContainer>
-        <StyledNavListContainer flex margin={{ horizontal: 'none', vertical: 'small' }} tag='nav'>
-          {newsNavListURLs.map((url, i) => {
-            return (
-              <NavListItem colorTheme={colorTheme} key={url} label={newsNavListLabels[i]} url={url} />
-            )
-          })}
-        </StyledNavListContainer>
-      </StyledFooterSection>
-      <StyledFooterSection
-        align='center'
-        className='small'
-        background={(colorTheme === 'light')
-          ? zooTheme.light.colors.background.default
-          : zooTheme.dark.colors.background.container
-        }
-        direction='row'
-        fill='horizontal'
-        pad={{ horizontal: 'large', vertical: 'none' }}
-        justify='between'
-        tag='section'
-      >
-        <StyledNavListContainer
-          direction='row'
-          gap='small'
-          responsive={false}
-          tag='nav'
-        >
-          {policyNavListURLs.map((url, i) => {
-            return (
-              <ThemeProvider key={url} theme={{ mode: colorTheme }}>
-                <StyledSmallNavListItem href={url}>
-                  {policyNavListLabels[i]}
-                </StyledSmallNavListItem>
-              </ThemeProvider>
-            )
-          })}
-        </StyledNavListContainer>
-        <div>
-          {props.adminContainer}
-          <StyledEasterEgg src='https://s3.amazonaws.com/zooniverse-static/assets/penguin.png' alt='' />
-        </div>
-      </StyledFooterSection>
-    </Box>
+            <Box
+              border={{
+                color: '#979797',
+                side: 'bottom',
+                size: 'xsmall'
+              }}
+              direction='row'
+              justify='between'
+              margin={{ bottom: 'medium' }}
+              pad={{ bottom: 'medium' }}
+            >
+              <LogoAndTagline
+                colorTheme={colorTheme}
+                tagLine={props.zooTagline}
+              />
+              <Box
+                align='end'
+                direction='row'
+                gap='small'
+                justify='end'
+                responsive={false}
+                tag='nav'
+              >
+                <SocialAnchor colorTheme={colorTheme} service='facebook' />
+                <SocialAnchor colorTheme={colorTheme} service='twitter' />
+                <SocialAnchor colorTheme={colorTheme} service='instagram' />
+              </Box>
+            </Box>
+
+            <Grid
+              fill
+              gap='medium'
```

Comment:
This gap seems a bit big to me. I believe the width Becky designed to is 1280px, so currently this is wrapping the columns at a much larger size (1380px). Maybe use `gap='small'` here to get closer to 1280?

Reply:
FWIW, Grommet's `large` size works out to `1152px` - if we're going for a maximum width, why don't we use that and adjust the `global.size` value in our theme?

Reply:
Maybe the theme sizes can be updated in a separate PR? I've been hesitant to do that because I doubt we can just change one.